packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -573,7 +569,7 @@ class CpuProfileData {
       if (tree.frameId == kRootId) {
         continue;
       }
-      (traceObject[CpuProfileData.traceEventsKey]! as List<Object?>).add({
+      (traceObject[_traceEventsKey]! as List<Object?>).add({
```

Comment:
should this be CpuProfileData._traceEventsKey for consistency?

Reply:
Done